src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -527,16 +539,15 @@ def _assess_jobs(self, all_jobs: list[BaseJob], all_clusters_by_id) -> list[JobI
 
             # Checking if Azure cluster config is present in cluster policies
             if cluster_config.policy_id:
-                try:
-                    policy = self._ws.cluster_policies.get(cluster_config.policy_id)
+                policy = self._safe_get_cluster_policy(cluster_config.policy_id)
+
+                if policy is not None:
```

Comment:
Reduce code nesting by: "If policy is None: continue"

Reply:
it should not continue otherwise we will not execute the rest of the code that does not depend on the policy, e.g. `if cluster_config.init_scripts: `